nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/nfeditor/jquery.nfeditor.js

```diff
@@ -236,6 +234,28 @@
             });
         },
 
+        /**
+         * Sets the size of the editor.
```

Comment:
Looks like a `CTRL-C`, `CTRL-V` here. Should update this to reflect what the function actually does 😄